UniversitiesApp/models.py

```diff
@@ -18,25 +18,50 @@ class University(models.Model):
     def __str__(self):
         return self.name
 
-    def save(self, *args, **kwargs):
-    	if not self.id:
-    		# Newly created object, so set slug            
-	    	original = slugify(self.name)	    	
-	    	#Checking if slug is unique
-	    	if University.objects.filter(slug=original).exists():	 
-	    		in_university = University.objects.get(slug=original)
-	    		#Make new slug by combining name and id of previous instance
-	        	self.slug = slugify('%s-%d' % (self.name, in_university.id))
-	        else:
-	        	self.slug = original
-
-        super(University, self).save(*args, **kwargs)
+    # def save(self, *args, **kwargs):
+    # 	if not self.id:
+    # 		# Newly created object, so set slug            
+	   #  	original = slugify(self.name)	    	
+	   #  	#Checking if slug is unique
+	   #  	if University.objects.filter(slug=original).exists():	 
+	   #  		in_university = University.objects.get(slug=original)
+	   #  		#Make new slug by combining name and id of previous instance
+	   #      	self.slug = slugify('%s-%d' % (original, in_university.id))
+	   #      else:
+	   #      	self.slug = original
+
+    #     super(University, self).save(*args, **kwargs)
```

Comment:
This is just sample code for assigning automatic unique slugs to universities.
It will be useful for future reference. Right now we have code that asks users to enter a unique slug to every university.